graphql/postWinners.js

```diff
@@ -0,0 +1,43 @@
+const firebase = require("../firebase.js");
+const db = firebase.db;
+const admin = firebase.admin;
+
+function postWinners(userHandles) {
+  var ref = db.collection("users");
+  userHandles.map(async user => {
+    const document = ref.doc(user);
+    var data = await document.get();
+    if (data["_createTime"] === undefined) {
```

Comment:
if (!data["_createTime"]) {

Reply:
not actually null, field actually says undefined